src/apps/watchfaces/OswAppWatchface.cpp

```diff
@@ -25,33 +27,34 @@ const char* OswAppWatchface::getAppName() {
 }
 
 #ifdef OSW_FEATURE_STATS_STEPS
-void OswAppWatchface::drawStepHistory(OswUI* ui, uint8_t x, uint8_t y, uint8_t w, uint8_t h, uint32_t max) {
+void OswAppWatchface::drawStepHistory(OswUI* ui, uint8_t posX, uint8_t posY, uint8_t width, uint8_t height, uint32_t max) {
     OswHal* hal = OswHal::getInstance();
     OswUI::getInstance()->resetTextColors();
     uint32_t weekDay = 0;
     uint32_t dayOfMonth = 0;
+    uint8_t index = 0;
```

Comment:
Why did you move this out of the `for`-loop if you are not accessing it outside the loop? Just leave it in there to reduce the potential of variable-name collisions.